test/e2e/cluster_size_autoscaling.go

```diff
@@ -32,7 +32,7 @@ const (
 	scaleDownTimeout = 30 * time.Minute
 )
 
-var _ = Describe("Autoscaling [Skipped]", func() {
+var _ = Describe("Cluster size autoscaling [Feature:Autoscaling]", func() {
```

Comment:
I don't want to run it as for now in Autoscaling suite. So how about changing the name to `[Feature:ClusterAutoscaling]`?